skdownscale/pointwise_models/utils.py

```diff
@@ -166,3 +166,17 @@ def ensure_samples_features(obj):
         if obj.ndim == 1:
             return obj.reshape(-1, 1)
     return obj  # hope for the best, probably better to raise an error here
+
+
+def check_datetime_index(obj, timestep):
+    """ helper function to check datetime index for compatibility
+    """
+    if isinstance(obj, pd.DataFrame):
```

Comment:
what should happen when obj is not a DataFrame?

Reply:
actually don't think we really need this function - I didn't end up using it in the NASA-NEX daily implementation and I think it's redundant with the testing that is in place now. seem reasonable @jhamman?

Reply:
Right. I don't think we need this anymore.

Reply:
Cool - just took this out.